src/utilities/mounted-ref.ts

```diff
@@ -0,0 +1,13 @@
+import {useRef, useEffect} from 'react';
+
+export function useMountedRef() {
+  const mounted = useRef(true);
+
+  useEffect(() => {
+    return () => {
+      mounted.current = false;
```

Comment:
Should this be true?? 🤔 
```suggestion
      mounted.current = true;
```

Reply:
Maybe I'm missing something?

Reply:
This one makes sense that it's false because it's for the clean-up?

Reply:
🤦‍♂ Didn't see it was in the returned function 😄